src/components/Contacts/ContactDetails/ContactDetailsTab/EditContactModal/EditContactModal.tsx

```diff
@@ -0,0 +1,696 @@
+      ...person.facebookAccounts.nodes.map((account) => ({
+        ...account,
+        type: 'facebook',
+        name: 'Facebook',
+      })),
+      ...person.twitterAccounts.nodes.map((account) => ({
+        ...account,
+        type: 'twitter',
+        name: 'Twitter',
+      })),
+      ...person.linkedinAccounts.nodes.map((account) => ({
+        ...account,
+        type: 'linkedin',
+        name: 'LinkedIn',
+      })),
+      ...person.websites.nodes.map((account) => ({
+        ...account,
+        type: 'website',
+        name: 'Website',
+      })),
+    ];
+    return socialAccounts.length > 0 ? (
+      <>
+        {socialAccounts.map((account, index) => (
+          <>
+            <ContactInputWrapper>
+              {index === 0 ? <ContactSocialIcon /> : null}
+              <Grid container spacing={3}>
+                <Grid item xs={6}>
+                  <TextField
+                    label={t('Username/URL')}
+                    value={account.value}
+                    fullWidth
+                  />
+                </Grid>
+                <Grid item xs={6}>
+                  <FormControl fullWidth>
+                    <InputLabel id="social-type-label">{t('Type')}</InputLabel>
+                    <Select
+                      labelId="socail-type-label"
+                      value={account.type}
+                      fullWidth
+                      readOnly
+                    >
+                      <MenuItem value={account.type}>{account.name}</MenuItem>
+                    </Select>
+                  </FormControl>
+                  <ContactEditDeleteIconButton>
+                    <DeleteIcon />
+                  </ContactEditDeleteIconButton>
+                </Grid>
+              </Grid>
+            </ContactInputWrapper>
+          </>
+        ))}
+      </>
+    ) : (
+      <>
+        <ContactInputWrapper>
+          <ContactSocialIcon />
+          <Grid container spacing={3}>
+            <Grid item xs={6}>
+              <TextField label={t('Username/URL')} value={null} fullWidth />
+            </Grid>
+            <Grid item xs={6}>
+              <FormControl fullWidth>
+                <InputLabel id="social-type-label">{t('Type')}</InputLabel>
+                <Select labelId="socail-type-label" value={null} fullWidth>
+                  <MenuItem value={'facebook'}>{t('Facebook')}</MenuItem>
+                  <MenuItem value={'twitterk'}>{t('Twitter')}</MenuItem>
+                  <MenuItem value={'linkedin'}>{t('LinkedIn')}</MenuItem>
+                  <MenuItem value={'website'}>{t('Website')}</MenuItem>
+                </Select>
+              </FormControl>
+              <ContactEditDeleteIconButton>
+                <DeleteIcon />
+              </ContactEditDeleteIconButton>
+            </Grid>
+          </Grid>
+        </ContactInputWrapper>
+      </>
+    );
+  };
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      title={'Edit People'}
+      content={
```

Comment:
Should have noticed this on your modal PR but could we just use `children` here instead? It would look more like nested content and would force all the modal related props to the top instead of some before and some after the giant content prop.

Reply:
That's a really good idea actually! I'll probably go ahead and do that in a separate pr. This one already has so much going on haha.